src/services/CohortService.ts

```diff
@@ -66,15 +66,94 @@ const deleteCohort = (cohortId: number) =>
     urlReplacements: { '{cohortId}': `${cohortId}` },
   });
 
+/**
+ * List module(s) for a cohort
+ */
+type MockModule = {
```

Comment:
MockCohortModule ?
Add a cohortId and a moduleId in here.
Should we be reusing the MockModule under src/types/Cohort.ts?

Reply:
I like the rename suggestion and oops I didn't mean to leave a redundant type definition 😅